### PR TITLE
Adding localization for Haitian Creole and Spanish (Puerto Rico)

### PR DESCRIPTION
**Pull request by alpita-masurkar**
I would like to add localization/ support for Haitian Creole (ht) and Spanish (Puerto Rico) (es-PR).
Please let me know if I need to do something more on this branch.
Thank you. :) 

_Note: The Haitian Creole translation was contributed by Steve Patrick Louis_

### CODE REVIEW

File: src/locale/ht.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// French (Canada) [fr-ca]
```
Comment (iamkun) [inline src/locale/ht.js:1]: wrong comment here

**Comment by iamkun** [comment]
THX

File: src/locale/ht.js (hunk 1 above)
Comment (alpita-masurkar) [reply src/locale/ht.js]: thanks. Fixed.

**Comment by amasurkar-verimatrix** [comment]
Thank you for quickly reviewing it @iamkun. Do you need anything else on this from me?

**Comment by alpita-masurkar** [comment]
Thank you for quickly reviewing this @iamkun. Do you need anything else on this from me?

**Pull request merged**